external/camel/README.md

```diff
@@ -0,0 +1,20 @@
+# External (Third Party Container) Tests
+Third Party container tests help verify that the AdoptOpenJDK binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries. For each application, we choose to run a selection of their functional tests.
```

Comment:
Please change the first part of the sentence to be:
Camel tests are part of the external third-party application tests that help verify that the Adoptium binaries are good...

and update references to AdoptOpenJDK to be Adoptium please.

Reply:
Done